lib/puppet/module_tool/skeleton/templates/generator/Gemfile

```diff
@@ -1,7 +1,14 @@
-source 'https://rubygems.org'
+source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 
 puppetversion = ENV.key?('PUPPET_VERSION') ? "#{ENV['PUPPET_VERSION']}" : ['>= 3.3']
+gem 'metadata-json-lint'
 gem 'puppet', puppetversion
 gem 'puppetlabs_spec_helper', '>= 0.8.2'
```

Comment:
Heh, it should be 1.0.0 but that's another issue

https://tickets.puppetlabs.com/browse/MODULES-1139